Steps4Impact/App/AppController.swift

```diff
@@ -55,6 +55,11 @@ class AppController {
     // Setup Telemetry
     AppEvents.activateApp()
 
+    // Customizing back button image
```

Comment:
Can you customize the navigation controller in the subclass instead of using .apperance `CommonUI/ViewControllers/NavigationController.swift`